documentation/catalog/guides/quickstart.md

```diff
@@ -230,37 +192,28 @@ just recreate
 ## Directory Structure
 
 ```text
-openverse-catalog
-├── .github/                                # Templates for GitHub
-├── archive/                                # Files related to the previous CommonCrawl parsing implementation
-├── docker/                                 # Dockerfiles and supporting files
-│   └── upstream_db/                        #   - Docker image for development Postgres database
-├── catalog/                                # Primary code directory
-│   ├── dags/                               # DAGs & DAG support code
-│   │   ├── common/                         #   - Shared modules used across DAGs
-│   │   ├── data_refresh/                   #   - DAGs & code related to the data refresh process
-│   │   ├── database/                       #   - DAGs related to database actions (matview refresh, cleaning, etc.)
-│   │   ├── maintenance/                    #   - DAGs related to airflow/infrastructure maintenance
-│   │   ├── oauth2/                         #   - DAGs & code for Oauth2 key management
-│   │   ├── providers/                      #   - DAGs & code for provider ingestion
-│   │   │   ├── provider_api_scripts/       #       - API access code specific to providers
-│   │   │   ├── provider_csv_load_scripts/  #       - Schema initialization SQL definitions for SQL-based providers
+
+catalog/                                # Primary code directory
+├── dags/                               # DAGs & DAG support code
+│   ├── common/                         #   - Shared modules used across DAGs
+│   ├── data_refresh/                   #   - DAGs & code related to the data refresh process
+│   ├── database/                       #   - DAGs related to database actions (matview refresh, cleaning, etc.)
+│   ├── maintenance/                    #   - DAGs related to airflow/infrastructure maintenance
+│   ├── oauth2/                         #   - DAGs & code for Oauth2 key management
+│   ├── providers/                      #   - DAGs & code for provider ingestion
+│   │   ├── provider_api_scripts/       #       - API access code specific to providers
+│   │   ├── provider_csv_load_scripts/  #       - Schema initialization SQL definitions for SQL-based providers
 │   │   │   └── *.py                        #       - DAG definition files for providers
 │   │   └── retired/                        #   - DAGs & code that is no longer needed but might be a useful guide for the future
-│   └── templates/                          # Templates for generating new provider code
-└── *                                       # Documentation, configuration files, and project requirements
+│   ├── templates/                          # Templates for generating new provider code
```

Comment:
The spacing on the comments will need to be adjusted so it all aligns on the right, do you mind adjusting that?
![image](https://github.com/WordPress/openverse/assets/10214785/8e5f5fc0-d500-4f91-9ae6-a14dd19ed164)